app/page.tsx

```diff
@@ -1,16 +1,26 @@
 import ScrollToTop from "@/components/ui/back-to-top";
 import { Hero } from "@/components/landing/hero";
 import { TabSection } from "@/components/landing/tabs";
+import { Navigation } from "@/components/landing/navigation";
 
 export default function Home() {
     return (
-        <main className='h-full overflow-auto'>
+        <main className='mx-auto h-full w-full overflow-clip bg-gradient-to-b from-[#493b6c] via-[#14243a] to-[#000000]'>
             <ScrollToTop />
-            <section className='h-screen w-full bg-gradient-to-b from-[#493b6c] via-[#14243a] to-[#000000]'>
+
+            <Navigation></Navigation>
```

Comment:
Use self closing tags like `<Navigation />` if this component doesn't take any parameters